lightblue-ldap-integration-test/src/test/resources/crud/insert/department-insert-template.json

```diff
@@ -8,6 +8,6 @@
         "objectClass": ["top", "groupofNames"],
         "cn": "#cn",
         "description": "#description",
-        "member": [#members]
+        "member": ["#members"]
     }
```

Comment:
That, or I suppose the unit tests could be changed to not include the surrounding quotation marks.